aioseop_class.php

```diff
@@ -919,6 +919,9 @@ public function get_title_format( $args ) {
 		$w            = $info['w'];
 		$p            = $info['p'];
 
+		if ( strpos( $title_format, '%site_title%' ) !== false ) {
```

Comment:
The parent class method for strpos should be used instead. (multiple occurances)

Reply:
https://github.com/semperfiwebdesign/all-in-one-seo-pack/blob/53b0a5f9c74941979d645caa4866145287f7e944/admin/aioseop_module_class.php#L302

Reply:
Done.